modules/bootstrapped/test/src/smithy4s/schema/DefaultValueSpec.scala

```diff
@@ -147,15 +147,16 @@ final class DefaultValueSpec extends FunSuite {
     testCaseOpt(Foo.f, None)
   }
 
+  test("nullable") {
+    val b: Schema[Nullable[Int]] = Schema.int.nullable
+    testCaseOpt(b, Some(Nullable.Null))
+  }
+
   private def testCaseOpt[A](schema: Schema[A], expect: Option[A])(implicit
       loc: Location
   ): Unit = {
     val sch = schema.addHints(smithy.api.Default(Document.DNull))
     val res = sch.getDefaultValue
     assertEquals(res, expect)
   }
-
-  private def testCase[A](schema: Schema[A], expect: A)(implicit
```

Comment:
Can we re-add tests with non-null documents ?

Reply:
sure thing :+1: